tests/phpunit/tests/test-wpml-config-locations.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+class WPML_Config_Locations_Test extends PLL_UnitTestCase {
+
+	protected static $dirs = array(
+		'themes' => array(
+			'best-theme',
+			'best-child',
+		),
+		'plugins' => array(
+			'best-plugin',
+		),
+	);
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		parent::wpSetUpBeforeClass( $factory );
+
+		// Copy themes and plugins from tests/phpunit/data to wp-content.
+		foreach ( self::$dirs as $type => $subdirs ) {
+			foreach ( $subdirs as $name ) {
+				$source_dir = dirname( __DIR__ ) . "/data/{$type}/{$name}";
+				$dest_dir   = WP_CONTENT_DIR . "/{$type}/{$name}";
+
+				@mkdir( $dest_dir );
+
+				foreach ( glob( $source_dir . '/*.*' ) as $file ) {
+					copy( $file, $dest_dir . '/' . basename( $file ) );
+				}
+			}
+		}
+	}
+
+	public static function wpTearDownAfterClass() {
+		parent::wpTearDownAfterClass();
+
+		// Remove previously copied themes and plugins from wp-content.
+		foreach ( self::$dirs as $type => $subdirs ) {
+			foreach ( $subdirs as $name ) {
+				$dest_dir = WP_CONTENT_DIR . "/{$type}/{$name}";
+
+				foreach ( glob( $dest_dir . '/*.*' ) as $file ) {
+					unlink( $file );
+				}
```

Comment:
Same as above.

Reply:
Done in ea605a1b10c3ee74c22befe862613e7e1da76f5c.